docs/en/ingest-management/elastic-agent/configuration/elastic-agent-kubernetes-autodiscovery.asciidoc

```diff
@@ -135,7 +135,7 @@ The policy generated by this configuration looks like:
 ----
 
 [discrete]
-== Autodiscover targeted Pods using hints
+== Hints (annotations) based autodiscover
 
 beta[]
```

Comment:
Is this a specific function to mark this as beta? 
I think it makes sense to put it also in the title `Hints (annotations) based autodiscover - [BETA]`

Reply:
That's an alias that adds a standard message within the text. So I would say it's ok where it is now. You can see https://www.elastic.co/guide/en/fleet/master/elastic-agent-kubernetes-autodiscovery.html for the result.